conf/einstein.config

```diff
@@ -0,0 +1,31 @@
+/*
+* ------------------------------------------------------------------
+*   Config file for nextflow pipelines for Novo Nordisk's Einstein cluster
+* ------------------------------------------------------------------
+*/
+
+params {
+    config_profile_description = "Novo Nordisk's Einstein cluster profile"
+    config_profile_contact = 'Ashot Margaryan (aoqn@novonordisk.com)'
+    max_memory = "750.GB"
+    max_cpus = 64
+    max_time = "240.h"
+}
+
+process {
+    executor = 'slurm'
+    scratch = true
+    maxRetries = 3
+    scratch = "/scratch/users/${USER}"
```

Comment:
Has this been tested? It may fail as the `${USER}` will not resolve witthin the config

Reply:
Users have access to only their own scratch directory at the moment, so that's why I want to keep the ${USER} or the equivalent option in the scratch path.

Reply:
> Thanks for the prompt look James! This works, as I guess the $USER is a unix variable. This can of course be overridden in some cases, so do you mean I should put something like `whoami` or `logname` instead?

Nope, I meant more that in some cases  groovy evaluates it as a literal string or it failed. But if it works here, all good :+1: